flixel/tile/FlxBaseTilemap.hx

```diff
@@ -730,19 +733,70 @@ class FlxBaseTilemap<Tile:FlxObject> extends FlxObject
 		{
 			return null;
 		}
-		
+
+		var zone:FlxTilemapZone = switch (PathingMethod)
+		{
+			// Basic pathing method where distances from starting point is calculated for all tiles
+			// until reaching the end point, and then path is found by backtracking from end point to starting point
+			case FlxTilemapPathingMethod.BASIC:
+				{x0:0, y0:0, sizeX:widthInTiles, sizeY:heightInTiles, indexStart:startIndex, indexEnd:endIndex};
```

Comment:
This probably leads to Dynamic objects in native targets. We should have classes that store data for FlxTilemapPathing types. That way we can create constructors that make our code look nice and pool them and reduce GC load.